.clang-format

```diff
@@ -17,10 +18,16 @@ AllowShortCaseLabelsOnASingleLine: false
 AllowShortFunctionsOnASingleLine: false
 AllowShortIfStatementsOnASingleLine: false
 AllowShortLoopsOnASingleLine: false
+AllowShortEnumsOnASingleLine: false
+AllowDesignatedInitializersOnASingleLine: false
 AlwaysBreakAfterReturnType: None
 PenaltyReturnTypeOnItsOwnLine: 100
 PenaltyBreakAssignment: 100
-PenaltyExcessCharacter: 10
+PenaltyExcessCharacter: 100
```

Comment:
let's reduce PenaltyReturnTypeOnItsOwnLine (~5-20) so it would be preferred method to split

Reply:
Done (squashed + force-pushed, as we're about to merge).